modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -138,6 +155,106 @@ public void tearDown() {
 		_logCaptures.forEach(LogCapture::close);
 	}
 
+	@Test
+	public void testExportCustomObjectEntryMultipleCompanies()
+		throws Exception {
+
+		String objectDefinitionName = RandomTestUtil.randomString() + "abc";
+
+		JSONObject jsonObject1 = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"domain", "able.com"
+			).put(
+				"portalInstanceId", "able.com"
+			).put(
+				"virtualHost", "www.able.com"
+			).toString(),
+			"headless-portal-instances/v1.0/portal-instances",
+			Http.Method.POST);
+
+		_company = _companyLocalService.getCompany(
+			jsonObject1.getLong("companyId"));
+
+		_user = UserTestUtil.addUser(_company);
+
+		_objectDefinition1 = ObjectDefinitionTestUtil.publishObjectDefinition(
+			"A" + StringUtil.toLowerCase(objectDefinitionName),
+			Collections.singletonList(
+				ObjectFieldUtil.createObjectField(
+					"Text", "String", true, true, null,
+					RandomTestUtil.randomString(), _OBJECT_FIELD_NAME, false)),
+			ObjectDefinitionConstants.SCOPE_COMPANY,
+			TestPropsValues.getUserId());
+
+		_objectDefinition2 = ObjectDefinitionTestUtil.publishObjectDefinition(
+			"A" + StringUtil.toUpperCase(objectDefinitionName),
+			Collections.singletonList(
+				ObjectFieldUtil.createObjectField(
+					"Text", "String", true, true, null,
+					RandomTestUtil.randomString(), _OBJECT_FIELD_NAME, false)),
+			ObjectDefinitionConstants.SCOPE_COMPANY, _user.getUserId());
+
+		JSONObject jsonObject2 = HTTPTestUtil.invokeToJSONObject(
+			null, _exportTaskEndpoint() + _objectDefinition1.getName(),
+			Http.Method.POST);
+
+		JSONObject jsonObject3 = HTTPTestUtil.invokeToJSONObject(
+			null,
+			"headless-batch-engine/v1.0/export-task/" +
+				jsonObject2.getString("id"),
+			Http.Method.GET);
+
+		try {
+			JSONAssert.assertEquals(
+				JSONUtil.put(
+					"className", "com.liferay.object.rest.dto.v1_0.ObjectEntry"
+				).put(
+					"errorMessage", ""
+				).put(
+					"executeStatus", "COMPLETED"
+				).toString(),
+				jsonObject3.toString(), JSONCompareMode.LENIENT);
+		}
+		catch (AssertionError error) {
+			Assert.fail(jsonObject3.getString("errorMessage"));
```

Comment:
[echo] PMD VIOLATION: /opt/dev/projects/github/liferay-portal/modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java:219 > Do not call Assert#fail improperly. (PMD Rule: AssertFailJUnitRule)

Reply:
Could you check @magjed4289 ?

Reply:
I have removed the try catch block